.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: node_js
+
+node_js:
+  - stable
+
+install:
```

Comment:
@descholar-ceo i would suggest you to use before_script instead of install, which run all the script needed before proceed